### ISSUE TITLE
False-positive: medsprava.com.ua

### ISSUE THREAD

**Issue by Alex-302** [AdguardTeam/DeadDomainsLinter#7]
```
{
    "medsprava.com.ua": {
        "info": {
            "domain_name": "medsprava.com.ua",
            "used_last_24_hours": true,
            "registered_domain": "medsprava.com.ua",
            "registered_domain_used_last_24_hours": true
        },
        "matches": []
    }
}
```

![image](https://github.com/AdguardTeam/DeadDomainsLinter/assets/8361299/414f0224-b62b-4a2c-a93c-afa6fb703129)

Subdomain best.medsprava.com.ua is working.

Second-level domain is not used
![image](https://github.com/AdguardTeam/DeadDomainsLinter/assets/8361299/ceccdf5c-38dd-4974-b7fb-9721ef0aba00)

**Comment by Alex-302** [comment]
Some torrents and video hosting sites use this approach to disguise, and rules for these "dead" sites may be removed. 

For example, `innal.top`

```
{
    "innal.top": {
        "info": {
            "domain_name": "innal.top",
            "used_last_24_hours": false,
            "registered_domain": "innal.top",
            "registered_domain_used_last_24_hours": true
        },
        "matches": []
    }
}
```
but seedoff2.innal.top is alive.

```
{
    "seedoff2.innal.top": {
        "info": {
            "domain_name": "seedoff2.innal.top",
            "used_last_24_hours": true,
            "registered_domain": "innal.top",
            "registered_domain_used_last_24_hours": true
        },
        "matches": []
    }
}
```

**Comment by ameshkov** [comment]
In all cases `registered_domain_used_last_24_hours` is `true` so they're not valid for removal.

I cannot reproduce this issue so probably it will be fixed alongside the next update.

**Comment by Alex-302** [comment]
Yep, not reproduced now.
